src/components/views/MyMilestones.jsx

```diff
@@ -364,72 +358,33 @@ class MyMilestones extends Component {
   }
 
   cancelMilestone(milestone) {
```

Comment:
When you started to work on this issue we did not have this proof system yet in place. I guess that rather than removing it for the time being cancelling milestone should stay as it was.

Reply:
Okay so I just need to remove the confirmation option for milestone cancellation?